agent/util-scripts/gold/pbench-copy-results/test-31.txt

```diff
@@ -322,6 +322,7 @@ config = ndk-test-1
 date = 2018-05-23T03:21:32
 rpm-version = 0.50-1g799ea025
 iterations = 1, 1
+timestamp = 1900-01-01T00:00:00.000000
```

Comment:
I think we should use a name for the option that is less ambiguous than `timestamp`: I suggest `tar-ball-creation-timestamp`.